exercise-1.js

```diff
@@ -380,60 +502,98 @@
                 "tech_stack" : null
             },
         ];
+    
+        let bio = devs.forEach(console.log(devs.name + " specializes in " + devs["tech_stack"]))
```

Comment:
That's the output I am getting :
 let bio = devs.forEach(console.log(devs.name + " specializes in " + devs["tech_stack"]))
                       ^

TypeError: undefined is not a function